python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -35,7 +35,7 @@
 import lsst.afw.table as afwTable
 import lsst.geom
 import lsst.meas.base as measBase
-from lsst.meas.base.fluxUtilities import FluxResultKey
+from lsst.meas.base import FluxResultKey
```

Comment:
If we are going to import `FluxResultKey` from measBase, we might as well not do it and use it `FluxResultKey` with the namespace specified wherever it is used.

Reply:
Yes, the code is already importing `measBase` so may as well use `measBase.FluxResultKey` in here and remove the second import completely.

Reply:
Implemented to `use measBase` tested locally against `meas_base` changes in 
https://github.com/lsst/meas_base/pull/233
as sell as the latest daily release.